review/fileToReview.js

```diff
@@ -1,6 +1,5 @@
 
 var domain = "bank.local.fr"
```

Comment:
```suggestion
cons domain = "bank.local.fr"
```

Utiliser const car cela permet de scoper la variable et de plus modifiser sa valeur, il faut aussi la mettre dans le scope de la fn `fetchTransactions`.